index.html

```diff
@@ -95,8 +96,90 @@ <h3>Featured Speakers</h3>
+            </div>
+          </div>
+          <div class="clearfix"></div>
+          <div class="main-speaker">
+            <div class="picture-speaker">
+              <img src="img/speakers/julien.png" alt="">
+            </div>
+            <div class="description-speaker">
+              <h4>Julien Paluku</h4>
+              <span>Minister, DRC Ministry of Industry</span>
+              <div></div>
+              <p>
+                He has been appointed Minister of Industry since April 12, 2021
+              </p>
+            </div>
+          </div>
+          <div class="clearfix"></div>
+          <div class="main-speaker">
+            <div class="picture-speaker">
+              <img src="img/speakers/malango.png" alt="">
+            </div>
+            <div class="description-speaker">
+              <h4>Malango Washikala</h4>
+              <span>Co-founder and Chief Executive Officer, Altech Group</span>
+              <div></div>
+              <p>
+                Social entrepreneur and development specialist
+              </p>
+            </div>
+          </div>
+          <div class="clearfix"></div>
+          <div class="main-speaker">
+            <div class="picture-speaker">
+              <img src="img/speakers/timo.png" alt="">
+            </div>
+            <div class="description-speaker">
+              <h4>Timo Ntoto Aley Angu</h4>
+              <span>Deputy Managing Director, Ecobank DRC</span>
+              <div></div>
+              <p>
+                Seasoned Banker with effective leadership and solid background in Corporate Finance
+              </p>
+            </div>
+          </div>
+          <div class="clearfix"></div>
+          <div class="main-speaker">
+            <div class="picture-speaker">
+              <img src="img/speakers/aaron.png" alt="">
             </div>
+            <div class="description-speaker">
+              <h4>Aaron Leopold</h4>
+              <span>Chief Executive Officer, EnerGrow</span>
+              <div></div>
+              <p>
+                Aaron is a thought and action leader in integrated and distributed energy policies, business models
+              </p>
+            </div>
+          </div>
+          <div class="clearfix"></div>
+          <div class="main-speaker">
+            <div class="picture-speaker">
+              <img src="img/speakers/atsuko.png" alt="">
+            </div>
+            <div class="description-speaker">
+              <h4>Atsuko Toda</h4>
+              <span>Director: Agriculture Finance and Rural Development Department, African Development Bank</span>
+              <div></div>
+              <p>
+                Experienced manager and expert
+              </p>
+            </div>
+          </div>
+          <div class="clearfix"></div>
+          <div class="more-speaker">
+            <button class="more-speaker-button">MORE  <img src="img/icon_down.svg" alt="Show more speakers"></button>
```

Comment:
- [ ] (OPTIONAL) Great job implementing the functionality of the `more` button. It would be great that when a user clicks the `more` button after the rest of the speakers have been displayed, a `less` button will appear so that when a user clicks that now, the number of speakers will go back to 2 👍 This will improve the user interface of your site.